parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ColumnChunkMetaData.java

```diff
@@ -341,6 +351,15 @@ public long getBloomFilterOffset() {
     return bloomFilterOffset;
   }
 
+  /**
+   * @return the length to the Bloom filter or {@code -1} if there is no bloom filter for this column chunk
+   */
+  @Private
+  public int getBloomFilterLength() {
```

Comment:
Return Optional<Int>?

Reply:
BTW, an e2e test case will be helpful to guarantee we have written the length correctly.

Reply:
it will be -1 by default and then length will be loaded from bloom filter header.
There are bunch of tests including E2E tests in TestParquetFileWriter, TestParquetMetadataConverter, ParquetRewriterTest, please take a look?